// @expo-google-fonts/newsreader/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const Newsreader_200ExtraLight = require('./200ExtraLight/Newsreader_200ExtraLight.ttf');
export const Newsreader_300Light = require('./300Light/Newsreader_300Light.ttf');
export const Newsreader_400Regular = require('./400Regular/Newsreader_400Regular.ttf');
export const Newsreader_500Medium = require('./500Medium/Newsreader_500Medium.ttf');
export const Newsreader_600SemiBold = require('./600SemiBold/Newsreader_600SemiBold.ttf');
export const Newsreader_700Bold = require('./700Bold/Newsreader_700Bold.ttf');
export const Newsreader_800ExtraBold = require('./800ExtraBold/Newsreader_800ExtraBold.ttf');
export const Newsreader_200ExtraLight_Italic = require('./200ExtraLight_Italic/Newsreader_200ExtraLight_Italic.ttf');
export const Newsreader_300Light_Italic = require('./300Light_Italic/Newsreader_300Light_Italic.ttf');
export const Newsreader_400Regular_Italic = require('./400Regular_Italic/Newsreader_400Regular_Italic.ttf');
export const Newsreader_500Medium_Italic = require('./500Medium_Italic/Newsreader_500Medium_Italic.ttf');
export const Newsreader_600SemiBold_Italic = require('./600SemiBold_Italic/Newsreader_600SemiBold_Italic.ttf');
export const Newsreader_700Bold_Italic = require('./700Bold_Italic/Newsreader_700Bold_Italic.ttf');
export const Newsreader_800ExtraBold_Italic = require('./800ExtraBold_Italic/Newsreader_800ExtraBold_Italic.ttf');
